### PR TITLE
Volumes in Regularization

### PR DESCRIPTION
**Pull request by lheagy**
- add back the volume term in the small, smooth contributions
- use the square root of the cell weights as the diag of the W matrix cc @fourndo

### CODE REVIEW

File: SimPEG/Regularization.py [hunk 1]
```diff
@@ -801,6 +751,62 @@ def mapping(self, val):
...
+            mesh=mesh, **kwargs
+        )
+
+    @property
+    def W(self):
+        """
+        Weighting matrix
+        """
+        if self.cell_weights is not None:
+            return Utils.sdiag(np.sqrt(self.cell_weights))
```
Comment (lheagy) [inline SimPEG/Regularization.py]: `SimpleSmall` uses the sqrt of the cell weights as the diag if they are included. There is no volume contribution.

File: SimPEG/Regularization.py [hunk 2]
```diff
@@ -951,6 +957,58 @@ def __init__(
...
+
+        super(Small, self).__init__(
+            mesh=mesh, **kwargs
+        )
+
+    @property
+    def W(self):
+        """
+        Weighting matrix
+        """
```
Comment (lheagy) [inline SimPEG/Regularization.py]: Small uses the sqrt of the cell_weights times the volume as the diag

File: SimPEG/Regularization.py [hunk 3]
```diff
@@ -1011,20 +1069,20 @@ def W(self):
         Weighting matrix that constructs the first spatial derivative stencil
         in the specified orientation
         """
-        W = getattr(
+        vol = self.regmesh.vol.copy()
+        if self.cell_weights is not None:
+            vol *= self.cell_weights
+
```
Comment (lheagy) [inline SimPEG/Regularization.py]: for the SmoothDeriv, we multiply be `sdiag(sqrt(Ave * (vol * cell_weights))`

Comment (fourndo) [inline SimPEG/Regularization.py]: Confirmed

File: SimPEG/Regularization.py (hunk 2 above)
Comment (fourndo) [inline SimPEG/Regularization.py]: Confirmed

**Pull request merged**